RecipeDetails/Views/WebViewController.swift

```diff
@@ -0,0 +1,126 @@
+        toolBar.translatesAutoresizingMaskIntoConstraints = false
+        return toolBar
+    }()
+    
+    private lazy var backButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(image: Images.RecipeDetails.chevronBackward, style: .plain, target: self, action: #selector(backwardAction))
+        button.isEnabled = false
+        return button
+    }()
+    private lazy var forwardButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(image: Images.RecipeDetails.chevronForward, style: .plain, target: self, action: #selector(forwardAction))
+        button.isEnabled = false
+        return button
+    }()
+    private lazy var shareButton = UIBarButtonItem(image: Images.RecipeDetails.share, style: .plain, target: self, action: #selector(shareAction))
+    private lazy var safariButton = UIBarButtonItem(image: Images.RecipeDetails.safari, style: .plain, target: self, action: #selector(safariAction))
+    private lazy var spacer = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: self, action: nil)
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.leftBarButtonItem = UIBarButtonItem(image: Images.RecipeDetails.chevronBackward, style: .plain, target: self, action: #selector(dismissThisVC))
+        navigationItem.rightBarButtonItem = UIBarButtonItem(image: Images.RecipeDetails.reload, style: .plain, target: self, action: #selector(reloadPage))
+        
+        loadRequest(url: url)
+        setupView()
+    }
+    
+    init(url: URL) {
+        self.url = url
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func loadRequest(url: URL) {
+        let urlRequest = URLRequest(url: url, cachePolicy: .returnCacheDataElseLoad, timeoutInterval: 5.0)
+        webView.load(urlRequest)
+    }
+    
+    private func setupView() {
+        view.addSubview(webView)
+        view.addSubview(toolBar)
+        
+        NSLayoutConstraint.activate([
+            webView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            webView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            webView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            
+            toolBar.topAnchor.constraint(equalTo: webView.bottomAnchor),
+            toolBar.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+            toolBar.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+            toolBar.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+        ])
+    }
+    
+    @objc private func dismissThisVC() {
+        dismiss(animated: true, completion: nil)
+    }
+}
+
+// MARK: - UIBarButtonItems @objc methods
+
+extension WebViewController {
```

Comment:
You could make `private extension` and there won't be any need to make every method `private`